tests/tests/smoke-tests.spec.js

```diff
@@ -25,7 +25,7 @@ context('Smoke Tests', () => {
       .should('be.visible');
 
     cy.getIframeBody()
-      .contains('Warnings')
+      .contains('All Messages')
```

Comment:
How do this work with Translations? What happens if translation changes? Do we need to adjust our tests?

Reply:
If translation changes the tests will fail - as far as I can remember we considered this a _good thing_, as text is the contract and if contract changes, the tests would fail.